code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -48,8 +48,8 @@
 //M4A3 PISTOL
 
 /obj/item/weapon/gun/pistol/m4a3
-	name = "\improper M4A3 service pistol"
-	desc = "An M4A3 Colt Service Pistol, the standard issue sidearm of the TerraGov Marine Corps. Uses 9mm pistol rounds."
+	name = "\improper C-22A service pistol"
```

Comment:
C45 uses .45 bullets, while the C22 uses 9mm.
Is the naming pattern based on caliber or on model number?
I'd stick with a single one, personally.

Reply:
Correct, on the pattern.

Will make the edit soon.